lib/Helper/Mail.php

```diff
@@ -229,7 +229,7 @@ public static function setAbsolutePaths(string $string, ?Model\Document $documen
 
         //matches all links
         preg_match_all("@(href|src)\s*=[\"']([^(http|mailto|javascript|data:|#)].*?(css|jpe?g|gif|png)?)[\"']@is", $string, $matches);
-        if (!empty($matches[0])) {
+        if ($matches[0]) {
```

Comment:
Should we move the preg_match_all in the if? This removes the count as integer or false.
Is the 0 key always set?

Reply:
i think we just can remove the IF and rely on `foreach($matches[0])`
Only case that might need to actually check if is existing is 
https://github.com/pimcore/pimcore/blob/072c9a52b83f8fcce609beb79edd7b7c7f1b2364/lib/Helper/Mail.php#L288-L293 due to `?string $css`

Reply:
@kingjia90 Okay, than we can remove the if because a foreach also work with an empty array.